lib/parsec/fileparse.py

```diff
@@ -265,9 +264,13 @@ def read_and_proc(fpath, template_vars=None, viewcfg=None, asedit=False):
 
     # process with EmPy
     if do_empy:
-        if flines and re.match('^#![Ee]m[Pp]y\s*', flines[0]):
+        if flines and re.match(r'^#![Ee]m[Pp]y\s*', flines[0]):
             if cylc.flags.verbose:
                 print "Processing with EmPy"
+
+            # import EmPy only when truly required (might throw ImportError)
+            from parsec.empysupport import EmPyError, empyprocess
```

Comment:
We might want to provide an explanation with this traceback so that it doesn't look like an internal error to the uninformed user:

```python
try:
    from parsec.empysupport import EmPyError, empyprocess
except ImportError as exc:
    exc.message = ('The empy Python module must be installed in '
                   'order to perform empy templating.\n%s' % exc.message)
    raise exc
```

Or

```python
raise ParsecError("he empy Python module...")
```